CAMERA_POSE: [-0.3449, 0.0190, 0.0130, -0.2925, -0.6215,  0.6756, -0.2673] #相机识别时的位置和四元数

MIDDLE_POSE: [-0.3549, -0.0012, 0.2292, 0.3167, 0.6104, -0.6516, 0.3198] #识别物体后过渡位置和四元数

CRAW_BEFORE: [0.5687, 0.4344, -0.3871, 0.5813] #识别后抓取时的四元数

SPEED: 20 #抓取时机械臂运动速度

CAMERA_ID: '150622072996' #右臂d435相机ID

AGV_TARGET_NAME: '7613B5D'
